cypress/page/iframe.page.ts

```diff
@@ -0,0 +1,27 @@
+/* eslint-disable require-jsdoc */
+class IframePage {
+  private urlPage: string;
+  private titleSelect: string;
+  private cssPagebutton: string;
+
+  constructor() {
+    this.urlPage = "https://www.w3schools.com/html/html_iframe.asp";
+    this.titleSelect = "[title='W3Schools HTML Tutorial']";
+    this.cssPagebutton = "a[title='CSS Tutorial']";
+  }
+  public visit(): void {
+    cy.visit(this.urlPage);
+  }
+
+  public getFrameTitle(): any {
+    return cy.get(this.titleSelect).its("0.contentDocument").its("body").then(cy.wrap).find("h1");
+  }
+
+  public goToCssPageInFrame(): any {
+    cy.get(this.titleSelect).its("0.contentDocument").its("body").then(cy.wrap).find(this.cssPagebutton).click();
+    return cy.get(this.titleSelect).its("0.contentDocument").its("body").then(cy.wrap)
+        .filter(":contains('CSS Tutorial')");
```

Comment:
the idea here is to use the also the iframe command to avoid this long concatenation of selectors. So, you can use the `iframe` command to enter to the HTML Tutorial iframe, and then find the CSS page. Also, you can add an additional command to verify the iframe now has the CSS tutorial open with the help of the `frameLoaded` command.